libnavigation-core/src/main/java/com/mapbox/navigation/core/accounts/MapboxNavigationAccounts.kt

```diff
@@ -0,0 +1,85 @@
+package com.mapbox.navigation.core.accounts
+
+import android.content.Context
+import android.text.format.DateUtils
+import com.mapbox.android.accounts.navigation.sku.v1.MauSku
+import com.mapbox.android.accounts.navigation.sku.v1.SkuGenerator
+import com.mapbox.android.accounts.navigation.sku.v1.TripsSku
+import com.mapbox.android.accounts.v1.AccountsConstants.MAPBOX_SHARED_PREFERENCES
+import com.mapbox.navigation.base.accounts.SkuTokenProvider
+
+// TODO: make the class internal
```

Comment:
What about making `navigationStopped` and `navigationStarted` `internal` instead? 🤔

Reply:
We can do that as well.

However, the class is public now only so that we can use it in the `examples` module. Same case for sessions and the trip service. We need to clean those examples up before the release.